src/rgw/cls_fifo_legacy.h

```diff
@@ -127,8 +127,8 @@ class FIFO {
   std::optional<marker> to_marker(std::string_view s);
 
   FIFO(lr::IoCtx&& ioc,
-       std::string oid)
-    : ioctx(std::move(ioc)), oid(oid) {}
+       const std::string& _oid)
+    : ioctx(std::move(ioc)), oid(_oid) {}
```

Comment:
i don't really agree with this change, and prefer the parameter names match their member variables

there certainly are cases where this shadowing can cause issues - for example, if you `std::move()` out of `foo` and need to refer to it later in the constructor - but you can still use `this->foo` to be explicit about it

i really don't there's enough risk to justify going through and changing existing code, especially for super-simple constructors like this one

Reply:
completely agree that we should not make such "style" changes to existing code.
made this change here only as a "tag along" to the other fixes.
however, if you think that the original code is better, I will revert.

Reply:
please revert

Reply:
done